test/screens/v2/candidate_generator/on_bus/departures_test.exs

```diff
@@ -0,0 +1,192 @@
+defmodule Screens.V2.CandidateGenerator.Widgets.OnBus.DeparturesTest do
+  use ExUnit.Case, async: true
+
+  alias Screens.Predictions.Prediction
+  alias Screens.Routes.Route
+  alias Screens.Stops.Stop
+  alias Screens.Trips.Trip
+  alias Screens.V2.CandidateGenerator.Widgets.OnBus.Departures
+  alias Screens.V2.Departure
+  alias Screens.V2.ScreenData.QueryParams
+  alias Screens.V2.WidgetInstance.Departures, as: DeparturesWidget
+  alias Screens.V2.WidgetInstance.Departures.NormalSection
+  alias Screens.V2.WidgetInstance.DeparturesNoData
+  alias Screens.V2.WidgetInstance.DeparturesNoService
+  alias ScreensConfig.Screen
+  alias ScreensConfig.V2.OnBus
+
+  import Mox
+  import Screens.Inject
+
+  @stop injected(Stop)
+
+  defp build_config() do
+    %Screen{
+      app_params: %OnBus{
+        evergreen_content: []
+      },
+      vendor: nil,
+      device_id: nil,
+      name: nil,
+      app_id: :on_bus_v2
+    }
+  end
+
+  defp build_departure(
+         route_id,
+         direction_id,
+         route_type \\ :bus,
+         arrival_time \\ ~U[2024-01-01 12:00:00Z]
+       ) do
+    %Departure{
+      prediction: %Prediction{
+        route: %Route{id: route_id, type: route_type},
+        trip: %Trip{direction_id: direction_id},
+        arrival_time: arrival_time
+      }
+    }
+  end
+
+  defp build_stop(id, options \\ %{connecting_ids: [], child_ids: [], parent_id: nil}) do
+    [
+      %Stop{
+        :id => id,
```

Comment:
I guess the formatter doesn't "fix" this but it shouldn't be necessary to use the long-hand map syntax in this case; like the other structs above, this can be `id: id` and etc. The only time when this should be required is for plain maps where the keys are not atoms (e.g. `%{"key" => "value"}`).